src/DependencyInjection/DoctrineFixturesExtension.php

```diff
@@ -19,14 +21,33 @@ class DoctrineFixturesExtension extends Extension
      * {@inheritDoc}
      *
      * @return void
+     *
+     * @throws Exception
      */
-    public function load(array $configs, ContainerBuilder $container)
+    public function load(array $configs, ContainerBuilder $container): void
     {
         $loader = new XmlFileLoader($container, new FileLocator(dirname(__DIR__) . '/../config'));
 
         $loader->load('services.xml');
 
+        $configuration = $this->getConfiguration($configs, $container);
+        $config        = $this->processConfiguration($configuration, $configs);
+
+        $container->setParameter('doctrine_fixtures.use_container_aware_loader', $config['use_container_aware_loader']);
```

Comment:
You're building a feature flag for a deprecated feature. That would only make sense if we instantly deprecated not opting out of the container-aware loader.